### PR TITLE
References

### PR DESCRIPTION
**Pull request by mwernimont**
Changes made:
-----------
Description

I chose the accordion style as when I am a user I like to control the amount of content shown at once.  I adjusted the chevrons to be SVGs.  I was finding it hard to use the SVGs as a background image in Vue but luckily found an article that went into the deep coolness that Sass allows us to do.  We can now control the color of the SVGs on both the left and down versions.  So choosing the colors of the accordions no longer have to depend on the chevrons static color.  

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/assets/images/chevronLeft.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 165 255"><path d="M165,36v4c-1.78,2.17-3.4,4.52-5.38,6.5q-38.06,38.16-76.18,76.25c-1.48,1.48-2.85,3.08-4.46,4.83,1.73,1.82,3.06,3.28,4.45,4.67q38.1,38.13,76.19,76.25c2,2,3.6,4.33,5.38,6.5v4c-1.55,2.12-2.86,4.47-4.68,6.32Q147.78,238.13,135,250.67a48.26,48.26,0,0,1-6,4.33h-4a78.69,78.69,0,0,1-7-5.58Q62.55,194.07,7.18,138.6C4.71,136.14,2.39,133.54,0,131v-7c1.57-1.71,3.07-3.49,4.71-5.13Q61.51,62,118.4,5.22A70.08,70.08,0,0,1,125,0h5q16.42,16.11,32.81,32.25C163.8,33.24,164.28,34.74,165,36Z" style="fill:#999b9e"/></svg>
```
Comment (collnell) [inline src/assets/images/chevronLeft.svg:1]: svgs are the best

File: src/components/References.vue [hunk 2]
```diff
@@ -1,30 +1,113 @@
...
+      </div>
     </div>
   </section>
 </template>
 
 <script>
     import referencesText from "@/assets/text/referencesText";
-
+    import USWDS from "uswds/src/js/components";
+    const { accordion } = USWDS;
```
Comment (collnell) [inline src/components/References.vue:34]: Did not know you could do this like that

File: src/components/References.vue [hunk 3]
```diff
@@ -1,30 +1,113 @@
...
+$icons:(
+  "chevronLeft": '<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 165 255"><path d="M165,36v4c-1.78,2.17-3.4,4.52-5.38,6.5q-38.06,38.16-76.18,76.25c-1.48,1.48-2.85,3.08-4.46,4.83,1.73,1.82,3.06,3.28,4.45,4.67q38.1,38.13,76.19,76.25c2,2,3.6,4.33,5.38,6.5v4c-1.55,2.12-2.86,4.47-4.68,6.32Q147.78,238.13,135,250.67a48.26,48.26,0,0,1-6,4.33h-4a78.69,78.69,0,0,1-7-5.58Q62.55,194.07,7.18,138.6C4.71,136.14,2.39,133.54,0,131v-7c1.57-1.71,3.07-3.49,4.71-5.13Q61.51,62,118.4,5.22A70.08,70.08,0,0,1,125,0h5q16.42,16.11,32.81,32.25C163.8,33.24,164.28,34.74,165,36Z" style="fill:%%COLOR%%"/></svg>',
+  "chevronDown": '<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 255 165"><path d="M35,0h6c2.14,1.83,4.42,3.52,6.41,5.51Q85.59,43.6,123.63,81.86c3.49,3.54,5.34,3.18,8.63-.14q38-38.34,76.24-76.34c2-2,4.32-3.59,6.5-5.38h5a30.15,30.15,0,0,1,4,2.9Q239.57,18.4,255,34v7c-1.33,1.48-2.6,3-4,4.43Q193.48,103,135.89,160.49A54.58,54.58,0,0,1,130,165h-5c-1.75-1.51-3.61-2.92-5.24-4.55Q62.39,103.14,5.11,45.72C3,43.58,1.68,40.59,0,38V37c1-1.89,1.65-4.15,3.09-5.6C13.64,20.84,24.34,10.45,35,0Z" style="fill:%%COLOR%%"/></svg>'
```
Comment (collnell) [inline src/components/References.vue:60]: also have not seen this yet, good find

File: src/components/References.vue (hunk 2 above)
Comment (mwernimont) [reply src/components/References.vue:34]: Yeah, saves us from loading in their whole package.  Just what we want, performance baby.

**Review by collnell — APPROVED** [review]
The sidebar is looking really good too!

**Pull request merged**